src/controlFlowCodingPractice_15March2023/ArrayListFeature.java

```diff
@@ -4,23 +4,43 @@
 import java.util.List;
 
 public class ArrayListFeature {
+    public void arrayListOperation() {
+        List<Integer> arrayList = new ArrayList<>();
+        arrayList.add(10);
+        arrayList.add(20);
+        arrayList.add(30);
+        arrayList.add(40);
 
-	public static void main(String[] args) {
-		// TODO Auto-generated method stub
-		List<Integer> arrayList = new ArrayList<>();
-		for (int i = 0; i <= 5; i++) {
-			arrayList.add(i);
-		}
-		System.out.println("Array List :" + arrayList);
-		// Remove elements from ArrayList
-		arrayList.remove(3);
-		System.out.println("Array List after remove element :" + arrayList);
+        System.out.println("Size of array list before remove elements :" + arrayList.size());
+        System.out.println("Array list before remove elements :" + arrayList);
 
-		// Print all elements from ArrayLis
-		for (int num : arrayList) {
-			System.out.print(num + " ");
-		}
+        // Remove elements from ArrayList
+        System.out.println("\nElement removed from arrayList is :" + arrayList.remove(1));
 
-	}
+        System.out.println("\nSize of array list after remove elements :" + arrayList.size());
+        System.out.println("Array list after remove element :" + arrayList);
```

Comment:
you are not checking the element is removed or not

Reply:
Done